.github/workflows/ci.yml

```diff
@@ -9,6 +9,9 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      actions: write
```

Comment:
I think we only need the pull-requests permission here, plus, move it within the scope of the step that needs it